### PR TITLE
fix(input): add emoji data source

### PR DESCRIPTION
**Pull request by lyjeileen**
## Change:
- Add dataSource props to allow people specify emoji data source 

## Screenshot
Screenshot 2024-12-08 at 11 55 07 AM

### CODE REVIEW

File: public/emoji/data.json
Comment (Shiti) [inline public/emoji/data.json:1]: why are we adding this file ? I think it should be used from whichever version of library is being used or specified url

File: src/components/input/baseInput/baseInput.tsx [hunk 1]
```diff
@@ -70,6 +68,7 @@ function BaseInputElement(
   const [isEmojiMenuShown, setIsEmojiMenuShown] = useState(false)
   const inputRef = useRef<HTMLInputElement>(null)
 
+  const database = new Database({ dataSource: props.emojiDataSource })
```
Comment (Shiti) [inline src/components/input/baseInput/baseInput.tsx:71]: `props.emojiDataSource` is optional but isnt the expected type string here ?

File: src/components/input/textInput/textInput.stories.tsx [hunk 2]
```diff
@@ -71,6 +71,7 @@ export const Default = {
     sender: { id: '169snlk9n', name: 'Some User' },
     conversationId: '1',
     placeholder: 'Type your message',
+    emojiDataSource: '/emoji/data.json',
```
Comment (Shiti) [inline src/components/input/textInput/textInput.stories.tsx:74]: Could we refer to the `node_modules` path instead ?

File: src/components/input/baseInput/baseInput.tsx (hunk 1 above)
Comment (lyjeileen) [reply src/components/input/baseInput/baseInput.tsx:71]: The emoji library always gives it a default value.

**Review by Shiti — APPROVED** [review]

**Pull request merged**